nginx/nginx.conf

```diff
@@ -177,7 +178,7 @@ http {
 
 rtmp {
     server {
-        listen 1935;
```

Comment:
Here you are removing the original listen statement, but not above. Can you add a comment in the config about why these need to be different?

Reply:
There's something odd going on with nginx that I can't quite explain.

If I add both `listen 1935` and `listen [::]:1935` to the config, nginx will refuse to start because "Address already in use". Having just `listen [::]:1935` gives this netstat output:

```
# netstat -tanp | grep 1935 | grep LIST
tcp6       0      0 :::1935                 :::*                    LISTEN      19/nginx: master pr
```

and I can connect on both IPv4 and IPv6:
```
# telnet 127.0.0.1 1935
Trying 127.0.0.1...
Connected to 127.0.0.1.
Escape character is '^]'.

# telnet localhost 1935
Trying ::1...
Connected to localhost.
Escape character is '^]'.
```

But, if I try the same thing with port 5000 and only have `listen [::]:5000`, I can *only* connect on the IPv6 address. the `listen 5000` statement is also needed to allow IPv4 connections. Also, weirdly nginx has no problem with this.

Any idea what's going on here?

Reply:
The RTMP feature is a nginx module, so maybe it has something to do with a difference there. If this works as is, I don't have a problem merging it. Just wanted to ensure this difference wasn't a mistake. I would add a comment so it doesn't get removed later because it seems redundant.